### ISSUE TITLE
can not obbtain login to bugs.python.org

### ISSUE THREAD

**Issue by luhler** [python/pythondotorg#955]
I sign-up, but never receive the E-mail link. There is nothing in the SPAM folder.

I want to report this bug with socket.py:

  File "C:\Python34\Lib\socket.py", line 53, in <module>
    from enum import IntEnum
ImportError: cannot import name 'IntEnum'

**Comment by berkerpeksag** [comment]
Please do not open duplicate issues. You are getting ImportError because Python tries to import the local enum.py file from your folder. Please remove or rename enum.py and the error will go away. See http://stackoverflow.com/questions/491705/python-problem-with-local-modules-shadowing-global-modules for more information.

**Comment by luhler** [comment]
I already renamed enum.py under Lib.
Under site-packages, I have an enum directory.  Here is the directory structure:
 Volume in drive C is Bobby_Upper
 Volume Serial Number is 60A5-51F8

 Directory of C:\Python34\Lib\site-packages

06/29/2016  10:21 AM    <DIR>          .
06/29/2016  10:21 AM    <DIR>          ..
06/26/2016  04:49 PM               267 easy-install.pth
05/21/2016  03:55 PM               126 easy_install.py
06/26/2016  05:21 PM    <DIR>          enum
06/26/2016  05:21 PM            29,842 enum34-1.1.1-py3.4.egg-info
06/26/2016  04:49 PM            72,520 enum34-1.1.6-py3.4.egg
06/29/2016  10:21 AM                 0 o.txt
06/18/2016  03:31 PM    <DIR>          pip
06/18/2016  03:31 PM    <DIR>          pip-8.1.2.dist-info
05/21/2016  03:55 PM    <DIR>          pkg_resources
06/14/2016  03:49 PM    <DIR>          pygame
08/03/2013  04:17 PM               121 README.txt
06/25/2016  08:23 AM    <DIR>          requests-2.10.0-py3.4.egg
05/21/2016  03:55 PM    <DIR>          setuptools
05/21/2016  03:55 PM    <DIR>          setuptools-18.2.dist-info
06/25/2016  09:14 AM    <DIR>          urllib3-dev-py3.4.egg
05/21/2016  03:55 PM    <DIR>          _markerlib
05/21/2016  03:55 PM    <DIR>          __pycache__
               6 File(s)        102,876 bytes
              13 Dir(s)  732,082,876,416 bytes free

```
On Wednesday, June 29, 2016 9:53 AM, Berker Peksag <notifications@github.com> wrote:
```

 Closed #955.—
